simapp/app.go

```diff
@@ -427,6 +428,20 @@ func (app *SimApp) SimulationManager() *module.SimulationManager {
 	return app.sm
 }
 
+func ConsensusParamsKeyTable() params.KeyTable {
```

Comment:
I think this can be made private
```suggestion
func consensusParamsKeyTable() params.KeyTable {
```

Reply:
Indeed.